src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.util.ArrayList;
+
+
+/**
+ * TaskList class is an extension of the java Array List class as it provides functionalities
+ * that are unique for duke.
+ */
+public class TaskList extends ArrayList<Task> {
+
+    public TaskList() {
+        super();
+    }
+
+    /**
+     * Function that prints all the tasks currently in the TaskList.
+     *
+     */
```

Comment:
Your JavaDoc is missing a `@return` tag and statement. Perhaps you could write it like this:
```suggestion
    /**
     * Function that prints all the tasks currently in the TaskList.
     *
     * @return The string containing all the tasks currently in the TaskList.
     */
```